examples/pod_exec_terminal_size.rs

```diff
@@ -0,0 +1,134 @@
+use futures::{channel::mpsc::Sender, SinkExt, StreamExt};
+use k8s_openapi::api::core::v1::Pod;
+
+use kube::{
+    api::{
+        Api, AttachParams, AttachedProcess, DeleteParams, PostParams, ResourceExt, TerminalSize,
+    },
+    Client, runtime::wait::{await_condition, conditions::is_pod_running},
+};
+use tokio::{io::AsyncWriteExt, select};
+use crossterm::event::{EventStream, Event};
+
+// send the new terminal size to channel when it change
+async fn handle_terminal_size(mut channel: Sender<TerminalSize>) -> Result<(), anyhow::Error> {
+    // get event stream to get resize event
+    let mut reader = EventStream::new();
+
+    let (width, height) = crossterm::terminal::size()?;
+    channel
+        .send(TerminalSize { height, width })
+        .await?;
```

Comment:
having tried this out a bit in two a terminal that i resized the window of a bit, i can't say i noticed any difference here. is there anything i should look out for?

Reply:
If I remember correctly, the width of the output wrapped around 80 characters or something by default. I think sending the width adjusts that.

Maybe try echoing long lines after adjusting the terminal width.

Reply:
I can see it working when cross-referencing with `pod_shell` now. It's subtle, only works when scaling above 80ch, and only affects new commands (it doesn't wrap what you've already received), but it totally works!